visualization.shadertoy/resources/language/resource.language.en_gb/strings.po

```diff
@@ -123,3 +123,39 @@ msgstr ""
 msgctxt "#30029"
 msgid "Channel 3 as *.png"
 msgstr ""
+
+msgctxt "#30030"
+msgid "Shadertoy Visualization"
+msgstr ""
+
+msgctxt "#30031"
+msgid "Incorrect reading of the given JSON file, check whether it is correct and available."
+msgstr ""
+
+msgctxt "#30032"
+msgid "Path to own preset list"
+msgstr ""
+
+msgctxt "#30033"
+msgid "Within here can be an own path defined to a preset list within JSON format. If you leave empty, addon use his own preset file."
+msgstr ""
+
+msgctxt "#30034"
+msgid "Define a single shader"
+msgstr ""
+
+msgctxt "#30035"
+msgid "Show help about preset file"
+msgstr ""
+
+msgctxt "#30036"
+msgid "This open a text viewer where gives help and how a preset file in JSON format need be defined."
```

Comment:
Open to read information on how-to build the JSON formatted preset file.

Reply:
"how to", not "how-to" (verb, not compound noun).

Reply:
Thanks a lot @chewitt and @ProfYaffle to look over it, has changed on all places to your ones.